circle.yml

```diff
@@ -7,7 +7,7 @@ machine:
     ADMINS: Olivier Le Floch <olefloch@work4labs.com>
     SECRET_KEY: EamJrjviiW13ue8PEamJrjviiW13ue8P
   python:
-    version: 2.7.9
+    version: 2.7.11
```

Comment:
They're all minor versions though?


On 18 mars 2016 19:37 -0700, Adrien Siebert<notifications@github.com>, wrote:
>  
> Incircle.yml(https://github.com/Work4Labs/django-short-urls/pull/40#discussion_r56742561):
>  
> >@@ -7,7 +7,7 @@ machine:>ADMINS: Olivier Le Floch<olefloch@work4labs.com>>SECRET_KEY: EamJrjviiW13ue8PEamJrjviiW13ue8P>python:>-  version: 2.7.9>+  version: 2.7.11
>  
> I would be very wary of upgrading more. I actually didn't notice the.9either, But our servers are still using2.7.6…
>  
>  
> —
> You are receiving this because you authored the thread.
> Reply to this email directly orview it on GitHub(https://github.com/Work4Labs/django-short-urls/pull/40/files/00c231bdd187204858d60da607eba38c050c41d5#r56742561)
>  
>  
>